### ISSUE TITLE
Some questions about the FRFT experiment

### ISSUE THREAD

**Issue by zzzikun** [tunakasif/torch-frft#16]
When I performed a classification task using FRFT with an order of 1, and compared it to applying FRFT twice with orders of 0.25 and 0.75, theoretically, the results of both methods should be the same. However, in my task, the latter achieved a 3% higher accuracy than the former. Why is that

**Comment by tunakasif** [comment]
There are two root causes: first, if you are using `frft()` (the continuous approximation) and not the `dfrft()` (actual discrete transform). Second, if you use it in a non-deterministic learning setting, convergence highly depends on the initialization. In both cases, the explanation is as follows:

1. Please remember that `frft()` is an approximation of the continuous transform, so the sum of transforms with different orders can deviate from the original more noticeably when compared to theory and `dfrft()`. The `dfrft()` will provide better order associativity, although you still need to keep in mind that you are dealing with finite precision. In other words, since we deal with complex numbers (`torch.complex64` in this repository), the elementary-wise difference between the sum of transforms with two orders and the transform with the sum of orders is in the range of `1e-7`. The total deviation is proportional to the number of elements. As you can verify from the tests, the equivalence tests use an `eps` value to check whether the difference is below a certain threshold as any sane equality check concerning floating numbers. Therefore, this difference may be the cause of your performance difference.
2. If you are not just using the transform but actually using it in a trainable environment, the training overall may also affect it. By default, the transform order is trainable, and the model performance is highly dependent on the initial transform order and its learning rate. You can set `trainable=False` (e.g., `DFrFTLayer(order=0.75, dim=-1, trainable=False)`) to have a static fractional Fourier transform layer.
